components/compliance-service/api/tests/test_ingest_to_mgr_conn.rb

```diff
@@ -26,6 +26,11 @@
     # those nodes should not have been added to the manual node manager, as they were ingested nodes, not manually added nodes
     manually_managed_nodes = MANAGER_GRPC nodes, :list, Nodes::Query.new(filters: [Common::Filter.new(key: "manager_id", values: ["e69dc612-7e67-43f2-9b19-256afd385820"])])
     assert_equal(original_manually_managed_nodes.total, manually_managed_nodes.total)
+
+    # ensure we can read an ingested node
+    nodes_list['nodes'].each { |node|
+      node_read = MANAGER_GRPC nodes, :read, Nodes::Id.new(id: node.id)
+    }
```

Comment:
It seems kind of strange to me that none of our other tests caught this. Do we not test ingested nodes anywhere else?

Reply:
ya i was surprised by that as well - 
we test compliance/reporting/nodes at the list and read level
we test nodes api at the list and read level for nodes added manually or via integration
but for nodes api + ingested nodes we were only testing list